introduction.md

```diff
@@ -0,0 +1 @@
+# I am Grigory and I am a person that likes to learn stuff.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.